packages/diagrams/frontend/sirius-components-diagrams/src/representation/operations.ts

```diff
@@ -35,6 +35,9 @@ export const diagramEventSubscription = gql`
             label
             description {
               id
+              ... on DiagramDescription {
+                autoLayout
+              }
```

Comment:
I may have not been explicit enough in the issue but this is what I'm trying to prevent.


> As such, the auto layout state of the diagram should be retrieve in DiagramRepresentation when the component is mounted.

I don't want to ask for the auto layout state every single time the diagram is refreshed. I want to retrieve the auto layout state the first time the representation is opened in `DiagramRepresentation`. Something like this:

```
const getDiagramDescriptionQuery = gql`
  query getDiagramDescription($editingContextId: ID!, $diagramId: ID!) {
    viewer {
      editingContext(editingContextId: $editingContextId) {
        representation(representationId: $diagramId) {
          description {
            ... on DiagramDescription {
              autoLayout
            }
          }
        }
      }
    }
  }
`
```

and 

```
export const DiagramRepresentation = () => {
  ...
  const { data, loading, error } = useQuery<..., ...>(getDiagramDescriptionQuery, { variables });
  ...
};
```

I don't want to receive this information every single time, just once when the diagram is opened.